pkg/controller/volume/attachdetach/cache/actual_state_of_world.go

```diff
@@ -685,11 +701,8 @@ func (asw *actualStateOfWorld) GetVolumesToReportAttachedForNode(nodeName types.
 
 	volumesToReportAttached := asw.getAttachedVolumeFromUpdateObject(nodeToUpdateObj.volumesToReportAsAttached)
 	// When GetVolumesToReportAttached is called by node status updater, the current status
```

Comment:
I think since we removed the logic here, the comment can be removed

Reply:
@jingxu97 I have updated this comment already. Could you please take an another look and inform if something is wrong with the comment.